karaf/apache-brooklyn/src/main/filtered-resources/etc/brooklyn.cfg

```diff
@@ -0,0 +1,24 @@
+################################################################################
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+################################################################################
+
+# Web login credentials
+
+# Credentials for user 'admin'
+brooklyn.webconsole.security.users=admin
```

Comment:
Do we want to have default credentials, I note the outcome from [[PROPOSAL] Remove unauthenticated localhost login](http://markmail.org/search/?q=[PROPOSAL]%20Remove%20unauthenticated%20localhost%20login#query:[PROPOSAL]%20Remove%20unauthenticated%20localhost%20login+page:1+mid:2f3q5vjf6rtxnpzt+state:results) was that we do not.

Reply:
Forgot this conversation, will change it back to no authentication